github/multiformats.yml

```diff
@@ -32,7 +32,6 @@ members:
     - dhruvbaldawa
     - dignifiedquire
     - djdv
```

Comment:
```suggestion
    - djdv
    - djmcquillan
```

Reply:
In other orgs we're following this pattern of moving people to `Alumni` team (doesn't exist here yet) rather than removing them from the org completely. 

I'm going to take care of this in a bit.